client/me/concierge/book/info-step.js

```diff
@@ -120,6 +135,9 @@ class InfoStep extends Component {
 							onChange={ this.setFieldValue }
 							value={ message }
 						/>
+						<FormSettingExplanation>
+							{ translate( 'Please respond in English.' ) }
```

Comment:
This should only appear if the notice does. It should not appear for `en` and `en-gb` users.

Reply:
Good point, updated.